Arion/include/Arion/ExpandingPolytopeAlgorithm.hpp

```diff
@@ -76,24 +73,33 @@ void BlowUpPolytope(arion::intersection::gjk::Simplex& simplex, ShapeA const& aS
         epona::HyperPlane const hyperPlane{
             simplex.vertices[0], simplex.vertices[1], simplex.vertices[2]
         };
+        glm::vec3 const& normal = hyperPlane.GetNormal();
 
-        glm::dvec3 const AB = simplex.vertices[1] - simplex.vertices[2];
-        glm::dvec3 const AC = simplex.vertices[0] - simplex.vertices[2];
-        glm::dvec3 const ABC = glm::cross(AB, AC);
+        glm::vec3 const a = cso::Support(aShape, bShape,  normal);
+        glm::vec3 const b = cso::Support(aShape, bShape, -normal);
 
-        glm::dvec3 const a = cso::Support(aShape, bShape, glm::normalize( ABC));
-        glm::dvec3 const b = cso::Support(aShape, bShape, glm::normalize(-ABC));
+        simplex.supportVertices[3] = hyperPlane.SignedDistance(a) > hyperPlane.SignedDistance(b)
```

Comment:
Could you add information about this logic change to commit message?

Reply:
That IsGreater call is replaced with operator more? Is this really necessary? Other than that there is no particular logic change, it's only the input range difference.

Reply:
Oh okay, I was just confused that you switched from AB/AC/ABC to hyperplane normal. Since I'm not familiar with these relations I was wondering if logic was changed somehow. Having this information in commit message would allow to find when some behavior was changed without `git blame`

Reply:
Well it basically does the same thing only with the help of another class. Previously it was calculating the plane normal using in place cross product, now it's done via hyper plane method. So the intended logic remains.